kafka_utils/kafka_consumer_manager/util.py

```diff
@@ -250,22 +255,22 @@ def remove_partition_from_consumer(self, partition):
         )
 
     def parse_consumer_offset_message(self, message):
-        key = bytearray(message.key)
+        key = message.key
         ((key_schema,), cur) = relative_unpack(b'>h', key, 0)
         if key_schema not in [0, 1]:
             raise InvalidMessageException()   # This is not an offset commit message
         (group, cur) = read_short_string(key, cur)
         (topic, cur) = read_short_string(key, cur)
         ((partition,), cur) = relative_unpack(b'>l', key, cur)
         if message.value:
-            value = bytearray(message.value)
+            value = message.value
             ((value_schema,), cur) = relative_unpack(b'>h', value, 0)
             if value_schema not in [0, 1]:
                 raise InvalidMessageException()  # Unrecognized message value
             ((offset,), cur) = relative_unpack(b'>q', value, cur)
         else:
             offset = None  # Offset was deleted
-        return str(group), str(topic), partition, offset
+        return group.decode(), topic.decode(), partition, offset
```

Comment:
The reason for the change here is that `str(b'123') == "b'123'"` in py3 (note the "b" in the string), but `str(b'123') == '123'` in py2, whereas `.decode()` does the right thing in both environments.

I find that `_tostr` function to be more confusing than enlightening, but I think that's a matter of preference.